.github/workflows/build_test_pr.yaml

```diff
@@ -2,8 +2,8 @@
 name: Build Dev Image
 on:
   pull_request:
-    branches:
-    - develop
+    branches-ignore:
+    - main
```

Comment:
Enable the standard dev workflow (i.e. build an image and run tests) on any branch that isn't `main`. (I was getting annoyed at making PRs to other branches and not having the tests running).

Reply:
Crap, looks like the image tag is invalid. I will fix that in a different PR.